Regex.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-useless-escape */
```

Comment:
__[HIGH]__  
This file is not needed at all. Users should implement it by their own.